fortran/plumed.f90

```diff
@@ -23,6 +23,92 @@ module plumed_module
   public :: plumed_f_gfinalize
   public :: plumed_f_gvalid
 
+  public :: plumed
+
+  type plumed
+    character(kind=c_char,len=32) :: handle
```

Comment:
I understand but at this level I prefer to avoid code duplication and base this interface on the other. I cannot do the other way round since not all compilers will support this interface.